src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -82,4 +82,16 @@ public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseExcepti
     public int numberOfUniquePrefixes() {
         return this.argMultimap.size() - 1;
     }
+
+    /**
+     * Checks if the {@code ArgumentMultimap} contains a valid entry for the specified prefix.
+     *
+     * @param prefix the {@code Prefix} we want to check for in the argument map.
+     * @return {@code True} if the prefix is present with exactly one non-empty argument;
+     *         {@code False} otherwise.
+     */
+    public boolean containsValidSinglePrefix(Prefix prefix) {
+        return argMultimap.containsKey(prefix) && argMultimap.get(prefix).size() == 1
+                && !argMultimap.get(prefix).get(0).isEmpty();
+    }
```

Comment:
I like the effectiveness of this code, can consider separating the boolean statements out however, for ease of reading 😄 

Also just want to check the necessity of the 3rd predicate since we already got the first?

Reply:
third predicate is necessary as the key value pair could be <" i/", [""]>, so we need to check if value associated with the prefix is empty or not.